src/server/pachyderm_test.go

```diff
@@ -10437,50 +10456,6 @@ func TestInterruptedUpdatePipelineInTransaction(t *testing.T) {
 	require.Matches(t, "outside of transaction", err.Error())
 }
 
-func TestPipelineSpecCommitCleanup(t *testing.T) {
```

Comment:
Is there a new corresponding test that makes sure we don't leave dangling spec repos? I might have missed it.

Reply:
Ah, that's a good point. I should come up with a good test for the system repo "cluster" deletion behavior, which I think is the equivalent guarantee now (no spec repo without its pipeline repo)